src/main/java/com/example/shorturl/controller/api/ShortenUrlController.java

```diff
@@ -39,15 +38,13 @@ public ShortenUrlController(final ShortenUrlService shortenUrlService){
     @PostMapping
     public ResponseEntity<UrlResponse> shortUrl(@RequestBody UrlRequest urlRequest) throws UrlNotFoundException {
 
-        if(Objects.isNull(urlRequest) || StringUtils.isEmpty(urlRequest.getUrl())){
+        if(Objects.isNull(urlRequest) || StringUtils.isEmpty(urlRequest.getUrl()))
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Url request is missing or empty");
-        }
 
         Optional<UrlEntity> optionalUrl = shortenUrlService.shortUrl(urlRequest);
 
-        if(optionalUrl.isEmpty()){
+        if(optionalUrl.isEmpty())
```

Comment:
UrlEntity entity = optionalUrl.orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Url is missing"));

Reply:
Fixed